.travis.yml

```diff
@@ -35,10 +35,10 @@ script:
     - mpirun -np 2 coverage run -p --source=mosfit -m mosfit -e SN2006le --travis -i 1 -f 1 -p 0
     - mpirun -np 2 coverage run -p --source=mosfit -m mosfit -e SN2006le.json --travis -i 1 --no-fracking -m magnetar
     - mpirun -np 2 coverage run -p --source=mosfit -m mosfit -e LSQ12dlf --travis -i 1 --no-fracking -m csm
-    - mpirun -np 2 coverage run -p --source=mosfit -m mosfit -e LSQ12dlf --travis -i 1 --no-fracking -m rprocess
+    - mpirun -np 2 coverage run -p --source=mosfit -m mosfit -e LSQ12dlf --travis -i 1 --no-fracking -m tde
     - coverage run -p --source=mosfit -m mosfit -e SN2007bg --travis -i 1 --no-fracking -m ic
     - coverage run -p --source=mosfit -m mosfit -e LSQ12dlg --travis -i 1 --no-fracking -m slsn -S 20 -E 10.0 100.0
-    - coverage run -p --source=mosfit -m mosfit -e 2010kd --travis -i 5 --no-fracking -m slsn --extra-bands u g --extra-instruments LSST -L 55540 55560 --exclude-bands B -s test --quiet -u
+    #- coverage run -p --source=mosfit -m mosfit -e 2010kd --travis -i 5 --no-fracking -m slsn --extra-bands u g --extra-instruments LSST -L 55540 55560 --exclude-bands B -s test --quiet -u
```

Comment:
Should be able to uncomment this line now that the code is able to merge with the master.